index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="styles.css" ></style>
+    </head>
+    <body>
+        <div class="phone-width">
+            <div class="app-bar">
+                <div class="bar">
+                    
+                </div>
+                <header class="toolbar">
+                    <div class="logo">
+                        <p>
+                            John Doe
+                        </p>
+                    </div>
+                    <menu class="union-vector">
+                        <img src="img\Normal Button\Tertiary\Icons\Union.png" alt="menu button">
+                    </menu>
+                </header>
+            </div>
+            <div class="headline">
+                
+                <p class="primary-text">Hey There. </br> I'm Alex</p>
+                
+                <p class="supporting-text">
+                    I'am a Software Developer
+                </p>
+                
+                <p class="description">
+                    I can help you build a product , feature or website Look through some of 
+                    my work and experience! If you like what you see and have a project 
+                    you need coded, don't hestiate to contact me.
+                </P>
+                <div class="decoration">
+                    <img src="img\Normal Button\Tertiary\Icons\Group2.png" alt="decoration">
+                </div>
+
+                <div class="social-media">
+                    <img src="img\Normal Button\Tertiary\Icons\Vector.png" alt="git-logo">
+                    <img src="img\Normal Button\Tertiary\Icons\Group.png" alt="linkdin-logo">
+                    <img src="img\Normal Button\Tertiary\Icons\Vector2.png" alt="V sign">
+                    <img src="img\Normal Button\Tertiary\Icons\Vector3.png" alt="twitter-logo">
+                    <img src="img\Normal Button\Tertiary\Icons\Vector4.png" alt="Letter M">
+                </div>
+
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Please implement it